src/ac_copilot_trainer/modules/ws_bridge.lua

```
@@ -15,6 +15,17 @@ M.PROTOCOL_VERSION = PROTOCOL_VERSION
 local pendingCoaching ---@type { lap: number, hints: table[], debrief: string|nil }|nil
 
+--- Round 10: per-corner LLM advisories keyed by corner label.
+--- Populated by pollInbound when corner_advice events arrive.
+local cornerAdvisories = {}  ---@type table<string, { lap: number, text: string, ts: number }>
+--- Round 10d: wall-clock reference updated via M.tick(simT). Replaces
+--- os.clock() which is process CPU time (advances too slowly for a
+--- low-CPU Lua script, so the 6s staleness expiry never triggered).
+local currentSimT = 0
+--- Per-corner last-query timestamp (unused after round 10c moved the
+--- debounce to realtime_coaching; kept for backward compat with M.reset).
+local lastCornerQueryAt = {}  ---@type table<string, number>
```

Luacheck (1.2.0)

[warning] variable 'lastCornerQueryAt' is never accessed

(W231)